torchci/pages/api/drci/drci.ts

```diff
@@ -0,0 +1,92 @@
+import type { NextApiRequest, NextApiResponse } from "next";
```

Comment:
Does this need to be an API? We can just run this stuff within a script instead of calling an API. That way you're not reliant on the server being up to be able to fetch the workflows.